pkgs/misc/uboot/default.nix

```diff
@@ -100,7 +117,6 @@ let
       homepage = "http://www.denx.de/wiki/U-Boot/";
       description = "Boot loader for embedded systems";
       license = licenses.gpl2;
-      maintainers = with maintainers; [ dezgeg samueldr lopsided98 ];
```

Comment:
Can you explain why  this change was needed? (And the resulting side-effects all over.)